chapter2_mdw/ExerciseTwo

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+chomp($radius = <STDIN>);
```

Comment:
Technically, we were supposed to prompt the user for the value, not just read the input.